### PR TITLE
Add metal ci job for baremetal-runtimecfg

### PR DESCRIPTION
**Pull request by cybertron**
We want to run the new metal ci job on this repo so we can easily
verify changes.

### CODE REVIEW

File: ci-operator/jobs/openshift/baremetal-runtimecfg/openshift-baremetal-runtimecfg-master-presubmits.yaml [hunk 1]
```diff
@@ -1,5 +1,82 @@
 presubmits:
   openshift/baremetal-runtimecfg:
+  - agent: kubernetes
+    always_run: true
```
Comment (celebdor) [inline ci-operator/jobs/openshift/baremetal-runtimecfg/openshift-baremetal-runtimecfg-master-presubmits.yaml]: Shouldn't this be:

```
always_run: false
```

Comment (cybertron) [reply ci-operator/jobs/openshift/baremetal-runtimecfg/openshift-baremetal-runtimecfg-master-presubmits.yaml]: Huh. I wonder if that got changed when I ran make jobs. I could swear I set it to false originally.

Comment (stbenjam) [reply ci-operator/jobs/openshift/baremetal-runtimecfg/openshift-baremetal-runtimecfg-master-presubmits.yaml]: If activity on baremetal-runtimecfg isn't running tons of tests per day, it  should be fine to turn it on.

**Comment by stbenjam** [comment]
https://github.com/openshift/release/pull/7813 should've fixed the instance problem reported in that run

/test pj-rehearse

**Comment by cybertron** [comment]
Cool, I was going to ask about that this morning.

/hold

Because I need to update the commit message if this is going in with always_run: true. Will wait for the results of this test run though.

**Comment by cybertron** [comment]
Looks like it just timed out. I think that's expected at this point?

**Comment by stbenjam** [comment]
Yea, timeout is normal, not all jobs succeeding and we've got to spend some time figuring out why things are so unstable.  Just resolve the merge conflict and I think this is good to go.

**Comment by cybertron** [comment]
/hold cancel

This should be good to go then. Thanks for all the feedback!

**Pull request merged**